kernel/src/callback.rs

```diff
@@ -52,22 +52,38 @@ impl AppId {
     }
 }
 
+/// Type to uniquely identify a callback subscription accross all drivers.
```

Comment:
```suggestion
/// Type to uniquely identify a callback subscription across all drivers.
```

Reply:
@gendx Looks like you didn't allow maintainers to push to PR branches. Could you apply this fix?

<img width="630" alt="image" src="https://user-images.githubusercontent.com/339422/66000488-5ff83a80-e454-11e9-8253-712a20cdcf8c.png">